cadc-web-token/src/main/java/org/opencadc/token/Assets.java

```diff
@@ -0,0 +1,177 @@
+ *
+ *  This file is part of the             Ce fichier fait partie du projet
+ *  OpenCADC project.                    OpenCADC.
+ *
+ *  OpenCADC is free software:           OpenCADC est un logiciel libre ;
+ *  you can redistribute it and/or       vous pouvez le redistribuer ou le
+ *  modify it under the terms of         modifier suivant les termes de
+ *  the GNU Affero General Public        la “GNU Affero General Public
+ *  License as published by the          License” telle que publiée
+ *  Free Software Foundation,            par la Free Software Foundation
+ *  either version 3 of the              : soit la version 3 de cette
+ *  License, or (at your option)         licence, soit (à votre gré)
+ *  any later version.                   toute version ultérieure.
+ *
+ *  OpenCADC is distributed in the       OpenCADC est distribué
+ *  hope that it will be useful,         dans l’espoir qu’il vous
+ *  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+ *  without even the implied             GARANTIE : sans même la garantie
+ *  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+ *  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+ *  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+ *  General Public License for           Générale Publique GNU Affero
+ *  more details.                        pour plus de détails.
+ *
+ *  You should have received             Vous devriez avoir reçu une
+ *  a copy of the GNU Affero             copie de la Licence Générale
+ *  General Public License along         Publique GNU Affero avec
+ *  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+ *  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+ *                                       <http://www.gnu.org/licenses/>.
+ *
+ *
+ ************************************************************************
+ */
+
+package org.opencadc.token;
+
+import ca.nrc.cadc.util.StringUtil;
+import org.json.JSONObject;
+
+import java.util.Objects;
+
+/**
+ * Class that represents the document that will be stored in cache.  Instances can be updated from time to time
+ * during a refresh.
+ */
+public final class Assets {
+    // Provide a short buffer to check for the expiry time.  This will be used to ensure that the expiry time isn't
+    // in the future by, say, one millisecond, which won't benefit a future request.  One minute is the default.
+    private static final long EXPIRY_BUFFER_CHECK_MS = 60000L;
+
+    // Keys to access the values in JSON.
+    static final String ACCESS_TOKEN_KEY = "access_token";
+    static final String REFRESH_TOKEN_KEY = "refresh_token";
+    static final String EXPIRES_IN_KEY = "expires_in";
+    private static final String EXPIRES_AT_MS_KEY = "expires_at_ms";
+
+
+    private final String accessToken;
+    private final String refreshToken;
+    private final long expiryTimeMilliseconds;
```

Comment:
I wonder if the subject and issuer (sub & iss) would be useful to have here for debugging or logging purposes.